ciphers/aes_enc_dec.py

```diff
@@ -0,0 +1,65 @@
+import sys, base64, json, math
+try:
+   from Crypto.Cipher import AES
+except ImportError:
+   print ("Error!! Module AES is required. Run either or following commands")
+   print("pip install pycrypto")
+   print("---------OR--------")
+   print("pip3 install pycrypto")
+   sys.exit()
+
+KEY=""
+PADDING_CHARACTER="S"
+VECTOR_FOR_AES="SUDESH1611GITHUB"
+
+def GetKey():
+    global KEY
+    tempKey = input("Enter password(min length: 8, max length: 32)")
+    while(len(tempKey.strip())<8 or len(tempKey.strip())>32 or ' ' in tempKey):
+        if(' ' in tempKey):
+            print("White spaces are not allowed!")
+        else:
+            print("Password must be at least 8 characters long and at max 32 characters long. Try Again!")
+    while(len(tempKey)%8!=0):
+        tempKey+=PADDING_CHARACTER
+    KEY=tempKey
+
+def AES_Encryption(cleartext):
+    if(len(KEY)<8 or len(KEY)%8!=0):
```

Comment:
```suggestion
    if len(KEY) < 8 or len(KEY) %8 != 0:
```
psf/black would make all these changes as discussed in CONTRIBUTING.md.